js/impress.js

```diff
@@ -724,24 +724,31 @@
         //   Well, the [tab] key by default navigates around focusable elements, so clicking
         //   it very often caused scrolling to focused element and breaking impress.js
         //   positioning. I didn't want to just prevent this default action, so I used [tab]
-        //   as another way to moving to next step... And yes, I know that for the sake of
-        //   consistency I should add [shift+tab] as opposite action...
+        //   as another way to moving to next step...
         document.addEventListener( "keyup", function( event ) {
 
+            // Handles Tab and Shift + Tab
+            if ( event.keyCode === 9 ) {
+              if ( event.shiftKey ) {
+                api.prev();
+              } else {
+                api.next();
+              }
+              return;
+            }
+
             if ( event.shiftKey || event.altKey || event.ctrlKey || event.metaKey ) {
```

Comment:
Please note that before any key was handled we check for `shirt`, `alt`, `meta` etc modifiers and don't do anything if they are used.

This is a fix for an issue for example with windows Alt+Tab or mac Cmd+Tab for switching windows that was causing slides to move.

With your change bug will be back - Alt+Tab and Cmd+Tab will again cause slides to move when switching windows. Before you check for tab or shift tab you should make sure no other modifier keys are pressed.